apps/src/music/blockly/toolbox/toolboxBlocks.ts

```diff
@@ -168,6 +168,12 @@ const toolboxBlocks: {[blockType in BlockTypes | string]: BlockInfoWithText} = {
     kind: 'block',
     type: BlockTypes.PLAY_SOUNDS_TOGETHER,
   },
+  [BlockTypes.PLAY_SOUNDS_TOGETHER_NO_NEXT]: {
+    levelbuilderText: 'Play Together',
```

Comment:
Update this to highlight the difference?

Reply:
Good call.  Updated.